### ISSUE TITLE
[React Native: Mac] AccessibilityActionInfo name component in mac is different from win32

### ISSUE THREAD

**Issue by rahjai** [microsoft/react-native-macos#411]
For win32 we use accessibilityStates and accessibilityActions for voiceover to announce correct state of the button i.e. collapsed or expanded.

We want to have similar behavior in mac, for which we currently use accessibilityState which does not work for mac and was looking to move to accessibilityStates to have the same accessibility functionality.

But below is the difference between win32 and mac
For win32
AccessibilityActionName = RN.AccessibilityActionName | 'AddToSelection' | 'RemoveFromSelection' | 'Select' | 'Expand' | 'Collapse';
For mac
AccessibilityActionName = RN.AccessibilityActionName;

Issue: mac is missing the remaining actions of win32 which is causing accessibility issues in mac
